### PR TITLE
CVA6 dual issue

### PR DESCRIPTION
**Pull request by cathales**

### CODE REVIEW

File: Project-Descriptions-and-Plans/cva6-dual-issue/PC_CVA6_Superscalar.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+
+- The project should not impact other configurations: each characteristic of
+  CVA6 without double-issue must be the same or better than before
+  modifications. Specifically, no area should be added at all if double-issue is
+  disabled.
+- The performance gain is not ISA-specific and does not require compiler
+  changes: the exact same code must be executed faster.
+- The project, as a CVA6 configuration option, will benefit from CVA6
+  verification work.
+
```
Comment (DBees) [inline Project-Descriptions-and-Plans/cva6-dual-issue/PC_CVA6_Superscalar.md:47]: Please state the TRL goal for this project

File: Project-Descriptions-and-Plans/cva6-dual-issue/PC_CVA6_Superscalar.md [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+- Thales
+- Tristan project (EU)
+
+## Initial Estimate of Timeline
+
+1. Fetch 64 bits (frontend): September 2023
+2. Double-issue (issue stage): October 2023
+3. Added ALU (EX stage): December 2023
+
```
Comment (DBees) [inline Project-Descriptions-and-Plans/cva6-dual-issue/PC_CVA6_Superscalar.md:91]: Do these initial estimates assume verification to TRL5?

File: Project-Descriptions-and-Plans/cva6-dual-issue/PC_CVA6_Superscalar.md [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+## Who would make use of OpenHW output
+
+Everyone interested in implementing a fast CVA6.
+
+Interested entities include:
+
+- Thales
+- Tristan project (EU)
+
```
Comment (DBees) [inline Project-Descriptions-and-Plans/cva6-dual-issue/PC_CVA6_Superscalar.md:85]: Can you say a little more about what application scenarios would like to use a dual issue CVA6?

File: Project-Descriptions-and-Plans/cva6-dual-issue/PC_CVA6_Superscalar.md [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+André Sintzoff, DIS
+
+### Project Manager, if a PM is designated
+
+None
+
+## Next steps/Investigation towards Project Launch (**PC only**)
+
+- Review of existing work on CVA6: September 2023
+
```
Comment (DBees) [inline Project-Descriptions-and-Plans/cva6-dual-issue/PC_CVA6_Superscalar.md:118]: Is there any software support to be scoped out by PL and resources allocated?
Is there additional verification support to be scoped out by PL and resources allocated?
Is the feature list relatively simple or is there work required to determine a feature list? (note the full feature list would come at PA, should have a rough plan and feature list by PL)

File: Project-Descriptions-and-Plans/cva6-dual-issue/PC_CVA6_Superscalar.md [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+## Target Date for PL
+
+End of September 2023
```
Comment (DBees) [inline Project-Descriptions-and-Plans/cva6-dual-issue/PC_CVA6_Superscalar.md:121]: ok that is an aggressive date, but doable I'd think. I'll put you on the agenda for Sep 25 TWG meeting

File: Project-Descriptions-and-Plans/cva6-dual-issue/PC_CVA6_Superscalar.md (hunk 1 above)
Comment (cathales) [reply Project-Descriptions-and-Plans/cva6-dual-issue/PC_CVA6_Superscalar.md:49]: I have updated this last item, is it okay?

File: Project-Descriptions-and-Plans/cva6-dual-issue/PC_CVA6_Superscalar.md (hunk 2 above)
Comment (cathales) [reply Project-Descriptions-and-Plans/cva6-dual-issue/PC_CVA6_Superscalar.md:94]: They do not. I think the modification for the comment above also answers this question.

**Review by jquevremont — APPROVED** [review]

**Comment by jquevremont** [comment]
@DBees, are we merging now or are we waiting for the results of the vote?

**Comment by DBees** [comment]
@jquevremont the flow we have used recently is to ballot first then merge. Let's keep to that for now until we discuss.

**Pull request merged**